src/dynamics/senh/dyn_comp.F90

```diff
@@ -114,7 +114,10 @@ subroutine dyn_readnl(NLFileName)
   use fvm_mod,        only: fvm_get_test_type
   use native_mapping, only: native_mapping_readnl
 #endif
-  use dimensions_mod_cam, only: qsize, qsize_d, ntrac, ntrac_d, npsq, ne, npart, lcp_moist
+#ifndef HOMME_WITHOUT_PIOLIBRARY
+  use mesh_mod,       only: MeshOpen
```

Comment:
I did not find the `mesh_mod.F90` code under the `senh` folder. This code does exist under the `se/dycore` folder. Does it mean the refined mesh capacity only works with CAM-SE dycore right now?

Reply:
Thanks for your clarification.